### PR TITLE
Validate jwt function

### PR DESCRIPTION
**Pull request by perarnborg**
Coses #75 

I decided to validate claims and header but not the signature of the jwt. Can be added later if we feel the need for this.

### CODE REVIEW

File: docs/guide/index.md [hunk 1]
```diff
@@ -41,6 +41,9 @@ const oAuthClient = createOauthClient({
   authorizationEndpoint, // Optional, defaults to '/authorize'
   tokenEndpoint, // Optional, defaults to '/token'
   tokenStorage, // Optional, defaults to 'localStorage'
+  tokenLeewaySeconds, // Optional, defaults to 60
```
Comment (dennerkrans) [inline docs/guide/index.md:44]: great argument name ✨

File: src/jwt.ts [hunk 2]
```diff
@@ -14,13 +29,13 @@ export const parseJwt = (token: string): Token => {
 
   return {
     header: parseJwtPart(tokenParts[0]),
-    payload: parseJwtPart(tokenParts.length > 1 ? tokenParts[1] : ''),
+    claims: parseJwtPart(tokenParts.length > 1 ? tokenParts[1] : ''),
     signature: tokenParts.length > 2 ? tokenParts[2] : ''
   }
 }
 
 // eslint-disable-next-line @typescript-eslint/no-explicit-any
```
Comment (dennerkrans) [inline src/jwt.ts:37]: shouldn't need this line anymore

**Review by dennerkrans — COMMENTED** [review]
Two comments. One is because good name, the other is because a comment can be removed :)

File: docs/guide/index.md (hunk 1 above)
Comment (perarnborg) [reply docs/guide/index.md:44]: That means a lot! 😸

**Pull request merged**